estimation/vision/calibration.hh

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <opencv2/opencv.hpp>
+
+//%deps(opencv)
+
+#include <vector>
+
+#include "sophus.hh"
+
+namespace estimation {
+namespace vision {
+
+
+struct CameraIntrinsics{
+	cv::Mat K;
+	cv::Mat D;
+};
+
+class CalibrationManager {
+ private:
+  std::vector<cv::Mat> all_camera_images;
+
+ public:
+  void add_camera_image(cv::Mat image);
+
+  int num_images_collected();
+
+  const CameraIntrinsics calibrate();
```

Comment:
Make this method const

Reply:
const method means 
`   const CameraIntrinsics calibrate() const;`

So the caller knows it doesn't mutate the class